packages/appium/lib/cli/extension-command.js

```diff
@@ -56,6 +57,20 @@ class ExtensionCommand {
     return this.config.extensionType;
   }
 
+  /**
+   * Logs a message and returns an {@linkcode Error} to throw.
+   *
+   * For TS to understand that a function throws an exception, it must actually throw an exception--
+   * in other words, _calling_ a function which is guaranteed to throw an exception is not enough--
+   * nor is something like `@returns {never}` which does not imply a thrown exception.
+   * @param {string} message
+   * @protected
+   * @returns {Error}
+   */
+  _createFatalError(message) {
```

Comment:
we already have method `error()` which does something entirely different.  so that's why it's not `_error()`.

the intent here is twofold:

1. the return value of this method should immediately be thrown
2. when the return value is thrown, the process should quickly exit. i.e. a "fatal" error

so... yeah.  how can I improve it?